app/views/pages/report.ejs

```diff
@@ -555,4 +737,276 @@
   var obfuscatePath = true;
 </script>
 
+<script>
+  /* Page validation functions. 
+  If validation fails and messages to be displayed should be dispatched in the validation function.
+  These aren't done */
+  const validateDemographicsAndGeography = () => {
+      //validate age
+      var hasAge = document.getElementById("Age").value != "";
+      if (!hasAge) {
+        return [false,"<%= __('Please select a age range') %>"];
+      }
+
+      //validate sex
+      var hasGender = radioSet(document.getElementsByName("gender"));
```

Comment:
I suggest calling it `isRadioSet` in order to indicate that it returns a boolean

Reply:
I've renamed the function and added a comment above it's definition to make it extra clear. This is my mistake. In my understanding of english radioSet implies that it returns a boolean although I can see now why it could cause confusion.